mods/ctf/ctf/teams.lua

```diff
@@ -437,6 +437,18 @@ minetest.register_on_joinplayer(function(player)
 	end
 end)
 
+function ctf.clearAssists(playerName)
```

Comment:
You would need to change all `playerName` instances to `player`.
```suggestion
function ctf.clearAssists(player)
```

Reply:
I have applied this.